src/Illuminate/Filesystem/Filesystem.php

```diff
@@ -149,9 +149,10 @@ public function replace($path, $content)
     /**
      * Prepend to a file.
      *
-     * @param  string  $path
-     * @param  string  $data
+     * @param  string $path
+     * @param  string $data
      * @return int
+     * @throws FileNotFoundException
```

Comment:
You need to add the fully qualified name of the exception. `\Illuminate\Contracts\Filesystem\FileNotFoundException`